README.md

```diff
@@ -33,3 +33,8 @@ e/c : increase/decrease only angular speed by 10%
 
 CTRL-C to quit
 ```
+
+# Twist with header
+Publishing a `TwistStamped` message instead of `Twist` can be enabled with the `stamped` parameter. Additionally the `frame_id` of the `TwistStamped` message can be set with the `frame_id` parameter.
+```
+ros2 run teleop_twist_keyboard teleop_twist_keyboard --ros-args -r -p stamped:=True -p frame_id:="base_link"
```

Comment:
Rather than document it like this, I think we should just have a section about `Parameters`.  Something like:

```suggestion
## Parameters
- `stamped (bool, default: false)`
  - If false (the default), publish a `geometry_msgs/msg/Twist` message.  If true, publish a `geometry_msgs/msg/TwistStamped` message.
- `frame_id` (string, default: '')`
  - When `stamped` is true, the frame_id to use when publishing the `geometry_msgs/msg/TwistStamped` message.
```